assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -277,6 +281,9 @@ che.infra.docker.master_api_endpoint=http://che-host:${CHE_PORT}/api
 # This is the webscoket base endpoint of the workspace master running within the core Che server.
 che.infra.docker.master_websocket_endpoint=ws://che-host:${CHE_PORT}/api/websocket
 
+# This is the minor webscoket base endpoint of the workspace master running within the core Che server.
+che.infra.docker.master_websocket_minor_endpoint=ws://che-host:${CHE_PORT}/api/websocket-minor
```

Comment:
It's not the best name to this property but I'm OK with that since docker infrastructure will be decommissioned